nirc_ehr/resources/queries/dbo/q_blood.sql

```diff
@@ -1,9 +1,15 @@
 
-SELECT anm.ANIMAL_ID_NUMBER AS Id,
-       CAST(anmEvt.EVENT_DATETIME AS TIMESTAMP) AS bloodDate,
+SELECT anmEvt.ANIMAL_EVENT_ID as objectid,
+       anm.ANIMAL_ID_NUMBER AS Id,
+       CAST(COALESCE (adt.CHANGE_DATETIME,anmEvt.EVENT_DATETIME) AS TIMESTAMP) AS bloodDate,
+       CAST(COALESCE (adt.CHANGE_DATETIME,anmEvt.EVENT_DATETIME) AS TIMESTAMP) AS modified,
        anmEvt.RESULT AS quantity,
-       anmCmt.TEXT AS remark
+       anmCmt.TEXT AS remark,
+       CASE WHEN adt.AUDIT_ID IS NULL THEN 'N'
```

Comment:
What's this isChanged used for? Validating the results via the schema browser? Is it stored anywhere? This is mostly my curiosity

Reply:
Yes, it was for validating purposes. Removed.